src/app/workflow/descriptors/descriptors.component.ts

```diff
@@ -48,10 +49,24 @@ export class DescriptorsWorkflowComponent extends EntryFileSelector {
     super(fileService, gA4GHFilesService, gA4GHService, filesService, filesQuery);
     this.published$ = this.workflowQuery.workflowIsPublished$;
   }
+
   getDescriptors(version): Array<ToolDescriptor.TypeEnum> {
     return this.descriptorService.getDescriptors(this._selectedVersion);
   }
 
+  getValidDescriptors(version): Array<any> {
+    return this.descriptorService.getValidDescriptors(this._selectedVersion);
+  }
+
+  /**
+   * Get all the primary or secondary descriptors
+   *
+   * @param {*} descriptor  This actually doesn't matter for the workflow components.
+   * Both tool and workflows uses the same abstract method, but only tool can have multiple descriptor types.
+   * Workflows won't use this until it also supports having multiple descriptor types.
```

Comment:
Fairly misleading comment that should probably be removed.  As:
1. This function is most definitely being used to retrieve file contents
2. Workflows in the GA4GH API can definitely have multiple descriptor types and requires a descriptorType as path parameter 

Though it does remind me that the container/workflow's code is completely identical and should probably be placed in the paramfilesService or descriptorService